gui/wxpython/gui_core/gselect.py

```diff
@@ -2370,6 +2370,17 @@ def getProjMatchCaption(projectionMatch):
                     )
                     data.append((layerId, raster, int(projectionMatch), grassName))
                     layerId += 1
+            elif self.dbWidgets["format"].GetStringSelection() == "Rasterlite":
+                rasters = self._getRasterliteDBRasters(dsn)
```

Comment:
I don't know anything about rasterlite, but that's a LOT of code just to list rasters. Can't GDAL be used for that? If it's really needed, maybe we should consider moving this somewhere else.

Reply:
> I don't know anything about rasterlite, but that's a LOT of code just to list rasters. Can't GDAL be used for that? If it's really needed, maybe we should consider moving this somewhere else.

That was the first solution I looked at, but for some reason I decided not to use `gdalinfo`. I looked again today and yes you are right the command `gdalinfo -json elevation.sqlite` is sufficient.

Reply:
Would `r.external` give you sufficient information? GDAL tools are not required dependency. If there is no tool in GRASS that would work, I am for using gdalinfo, but there would have to be better error handling in case users don't have gdal tools installed.

Reply:
Maybe @rouault has a suggestion here?

Reply:
> Maybe @rouault has a suggestion here?

if you can use the GDAL python bindings:
```
ds = gdal.Open("my.rasterlite")
ds.GetSubDatasets()
```
cf https://gis.stackexchange.com/questions/72178/how-to-extract-subdataset-from-hdf-raster

Reply:
> Would `r.external` give you sufficient information? GDAL tools are not required dependency. If there is no tool in GRASS that would work, I am for using gdalinfo, but there would have to be better error handling in case users don't have gdal tools installed.

`r.external` is not enough. Better error handling if Python `gdal` module is missing is fixed with https://github.com/OSGeo/grass/pull/2513/commits/da15c4eda7bb80ce22e7f7806d8e11a592ebd76c.

Reply:
> if you can use the GDAL python bindings:

Yes of course. Thanks.

Reply:
Looks like there is not a great solution here since neither gdal tools nor gdal python bindings are a GRASS dependency. I don't think python bindings are necessarily better solution than using gdal tools. Should we perhaps look at why we can't get the needed info from the existing grass tools that use gdal library? Maybe we can implement something there. Or maybe not, but I can't help with the decision without understanding this little more. 
Generally, it would be helpful and more effective to include some background information and reasoning behind the changes.

Reply:
What kind of information we need:

1. Get raster or rasters names from the Rasterlite DB.

I didn't found any GRASS GIS module (import or raster group) which was useful, that's was my first step before created this PR and  that's why I originally decided to use `db.select` module with SQL https://github.com/OSGeo/grass/pull/2513/commits/20b283052c3934be08166acb6ccf32e2d8e583ae.